handbook/company/why-this-way.md

```diff
@@ -48,6 +48,8 @@ There are three reasons for visiting [the docs](https://fleetdm.com/docs):
 - **Committed learning**: "I've decided to learn this. I need a curriculum to get me there; with content that makes it as easy as possible, surface-level as possible. I want to learn how Fleet works and how to do all the things."
 - **Quick reference**: "Is this thing broken or am I using it right? How do I use this?" Whether they just stumbled in from a search engine, an on-site search, or through the Fleet website navigation, visitors interested in quick reference are interested in getting to the correct answer quickly.  Quick referencers search for REST API pages, the config surface of the Fleet server, agent options, how to build YAML for `fleetctl apply`, the built-in MDM profiles, the table schema, the built-in queries, reference architectures and cost calculators for deploying your own Fleet instance.
 
+> Currently, Fleet is only adding "Quick reference" to the [docs](fleetdm.com/docs). New "Committed learning" docs should be written as [articles](https://fleetdm.com/articles). Why? The docs surface area is large and hard to maintain. We've learned that users are more forgiving when articles are out-of-date.
```

Comment:
Small suggestion to spin this in a more positive way:

```suggestion
> Currently, Fleet is only adding "Quick reference" to the [docs](fleetdm.com/docs). New "Committed learning" docs should be written as [articles](https://fleetdm.com/articles). Why? The docs surface area is large and Fleet iterates on features more quickly than we can update how-tos. We've learned that users are more forgiving when articles are out-of-date.
```

Reply:
Love it